integration_tests/test_commit_format.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Test the commit message format."""
+
+import os
+
+from utils import get_cmd_output
+
+COMMIT_TITLE_MAX_LEN = 50
+COMMIT_BODY_LINE_MAX_LEN = 72
+BASE_BRANCH = os.environ['BUILDKITE_PULL_REQUEST_BASE_BRANCH']
+
+
+def test_commit_format():
+    """
+    Checks commit message format for the current PR's commits.
+
+    Checks if commit messages follow the 50/72 git commit rule
```

Comment:
nit: after the explanation, add a link to the 50/72 rule so it doesn't look like they're just arbitrary numbers we happen to like

Reply:
Wanted to, then I forgot :(, thanks.